web/src/containers/Sidebar.js

```diff
@@ -200,28 +196,24 @@ class Sidebar extends Component {
     const hasImage = ['as', 'gu', 'mp', 'vi'].indexOf(place.id) < 0;
 
     return (
-      <div className="ds-l-col--3">
-        <aside className="site-sidebar">
-          <div className="site-sidebar__sticky">
-            <div className="ds-u-display--flex ds-u-align-items--center ds-u-border-bottom--1 ds-u-padding-y--2 ds-u-margin-bottom--4">
-              {hasImage && (
-                <img
-                  src={`/static/img/states/${place.id}.svg`}
-                  alt={place.name}
-                  className="ds-u-margin-right--2"
-                  width="40"
-                  height="40"
-                />
-              )}
-              <h1>{place.name}</h1>
-            </div>
-            <VerticalNav
-              selectedId={activeSection || 'apd-state-profile-overview'}
-              items={links}
-            />
+      <aside className={`site-sidebar${isIE() ? ' site-sidebar__ie' : ''}`}>
```

Comment:
I think we can just do...
```suggestion
      <aside className="site-sidebar">
```